### PR TITLE
[10.0][ADD] mrp_bom_comparison

### PR DESCRIPTION
**Pull request by sebalix**
Compare two Bill of Materials to view the differences.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Isn't better name `mrp_bom_comparison`?

**Comment by sebalix** [comment]
Yes, for the end user it is surely better, I'll do that.

**Comment by sebalix** [comment]
Module renamed to `mrp_bom_comparison`.

**Comment by pedrobaeza** [comment]
Runbot error is due to the current lack of README file. Maybe you can add the minimal README with the placeholder that will be later replaced.

File: mrp_bom_comparison/wizards/mrp_bom_comparison.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    _description = "BoM line difference"
+
+    wiz_id = fields.Many2one('wizard.mrp.bom.comparison', u"Wizard")
+    product_id = fields.Many2one('product.product', u"Product")
+    bom1_qty = fields.Float(
+        u"v1-Qté", digits=dp.get_precision('Product Unit of Measure'))
+    bom2_qty = fields.Float(
+        u"v2-Qté", digits=dp.get_precision('Product Unit of Measure'))
+    diff_qty = fields.Float(
+        u"Écart qté", digits=dp.get_precision('Product Unit of Measure'))
```
Comment (LoisRForgeFlow) [inline mrp_bom_comparison/wizards/mrp_bom_comparison.py]: You should put this in English and then add translations.

**Review by LoisRForgeFlow — COMMENTED** [review]
Interesting proposal, one quick comment about translations:

File: mrp_bom_comparison/wizards/mrp_bom_comparison.py (hunk 1 above)
Comment (sebalix) [reply mrp_bom_comparison/wizards/mrp_bom_comparison.py]: Thank you! This is a old module written for 6.0 a long time ago, I hope all terms are fixed now.

**Comment by sebalix** [comment]
@pedrobaeza I'm not sure how to deal with the README.rst. Is it still required even with the new `./readme` template directory?

**Comment by sebalix** [comment]
Nevermind, runbot seems OK now!

**Comment by pedrobaeza** [comment]
Yeah, that README is more than enough. In fact, you only need a minimal README with the placeholder for not having the warning (that any way will disappear when the README is generated).

**Review by StefanRijnhart — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Comment by sebalix** [comment]
Thank you all!

**Pull request merged**